cli/package.json

```diff
@@ -36,7 +36,7 @@
     "@ledgerhq/hw-transport-mocker": "5.51.1",
     "@ledgerhq/hw-transport-node-hid": "5.51.1",
     "@ledgerhq/hw-transport-node-speculos": "^5.51.1",
-    "@ledgerhq/ledger-core": "beta",
+    "@ledgerhq/ledger-core": "7.0.0-beta.4",
```

Comment:
we shouldn't bump somethng that is not yet prebuild released.

- https://github.com/LedgerHQ/lib-ledger-core-node-bindings/runs/2620426034 is failing
- it's not yet published https://github.com/LedgerHQ/lib-ledger-core-node-bindings/releases

these two actions are required for prebuild to be fetchable. (otherwise we enter unexpected bugs if there are no prebuilds)